### DISCUSSION TITLE
Using the examles from confetti in angular

### DISCUSSION THREAD

**Discussion by Webrow** [matteobruni/tsparticles#4955] [Q&A] [answered]
I am having troubles getting the examples from confettijs to run. (I want to use the realistic cannon). 
The confetti function is a reference to something, and I cant figure out what. 
```
const count = 200,
  defaults = {
    origin: { y: 0.7 },
  };

function fire(particleRatio, opts) {
  confetti(
    Object.assign({}, defaults, opts, {
      particleCount: Math.floor(count * particleRatio),
    })
  );
}

fire(0.25, {
  spread: 26,
  startVelocity: 55,
});

fire(0.2, {
  spread: 60,
});

fire(0.35, {
  spread: 100,
  decay: 0.91,
  scalar: 0.8,
});

fire(0.1, {
  spread: 120,
  startVelocity: 25,
  decay: 0.92,
  scalar: 1.2,
});

fire(0.1, {
  spread: 120,
  startVelocity: 45,
});
```

I use the angular setup for tsParticles.  I now use the "confetti" preset, which doesnt seem to work, until I use LoadConfettiPreset(engine).

**Comment by matteobruni** [comment, marked as answer]
I've created a sample using `tsparticles-confetti` in Angular here: https://codesandbox.io/s/charming-khorana-h7oei4?file=/src/app/app.component.ts

I'll close this here, if you need more help please use the Angular repository here: https://github.com/tsparticles/angular
